docs/manual/java/guide/build/EclipseSbt.md

```diff
@@ -0,0 +1,57 @@
+# Importing an sbt project into Eclipse
+
+If you worked through the command line example for [[sbt|GettingStartedSbt]], you have an sbt build. The sbt tool provides an [sbt-eclipse](https://github.com/typesafehub/sbteclipse) plugin that generates Eclipse project artifacts, which you can then import into Eclipse.
+
+Follow these steps to integrate your project with Eclipse:
+
+1. Configure sbt to include the Eclipse plugin:
+    
+    1. In the `project` folder of your sbt build, create an `eclipse.sbt` file. For example, with a project named `hello`, create a `hello/project/eclipse.sbt` file. 
+    
+    1. Add the following line to the file:
+    ```
+    addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
+    ```
+    1. Save the file. 
+    
+1. In a console, `cd` to the top-level folder, and enter `sbt` to start the build tool.
+
+    Continuing with the example of a `hello` project: 
+
+    ```
+    cd hello
+    sbt
+    ... (booting up)
+    >
+    ```
+    The `>` prompt tells you that sbt is running.
+
+1. Enter `eclipse` to generate the Eclipse project files for all projects in your build:
```

Comment:
This comment is WRT Ignasi's comment below. EclipseScala is obsolete, replaced by EclipseSbt.md in the build folder. Thanks for catching this, I've cleaned it up and it will be in my next commit.